app/presenters/pin_presenter.rb

```diff
@@ -21,7 +21,7 @@ def initialize(opts = {})
             else
               # NOTE: if we want to do by_user_gender, we'd do it here
               # but there's not an easy way to let people change this on the fly
-              Pin.includes(:user, :pin_images, :procedure, :surgeon).recent.paginate(:page => @page)
+              Pin.includes(:user).recent.paginate(:page => @page)
```

Comment:
I just happened to notice these were unneeded in the logs. Didn't track how much of a speed difference it makes on page load...